latex_access/tests/test_settings.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import latex_access
+from latex_access import settings as settings_module
+
+try:
+    settings_module.settings['brailletable'] = 'nemeth'
+    settings_module.brailleTableToUse()
+    settings_module.settings['brailletable'] = 'ueb'
+    settings_module.brailleTableToUse()
+    incompatible = False
+except AttributeError:
+    incompatible = True
+
+class TestSettings(unittest.TestCase):
+    def test_load_settings(self):
+        """Tests that settings are read from a file."""
+        settings_module.settings = {}
+        settings_module.loadSettings('latex_access/tests/test_settings.txt')
```

Comment:
Where possible it would be nice not to rely on exact location of these tests in the file system (we would probably need to move them at some point when we would get around to testing the scripts placed at the top level of the repo). You can get the directory where this test is located with `os.path.abspath(os.path.dirname(__file__))` and use `os.path.join` to create a complete path to the sample config.